Labs/BestFriendsIntegrityUnitID 3/Skorohoda/Lab2/src/main/java/com/example/demo/controller/TeamController.java

```diff
@@ -0,0 +1,79 @@
+package com.example.demo.controller;
+
+import com.example.demo.constants.HttpStatuses;
+import com.example.demo.dto.TeamDTO;
+import com.example.demo.dto.TeamRequestDTO;
+import com.example.demo.mapper.teamMapper.TeamMapper;
+import com.example.demo.service.TeamService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/team")
+public class TeamController {
+    private final TeamService teamService;
+    private final TeamMapper teamMapper;
+
+    public TeamController(TeamService teamService, TeamMapper teamMapper) {
+        this.teamService = teamService;
+        this.teamMapper = teamMapper;
+    }
+
+    @ApiOperation(value = "Find all teams")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = HttpStatuses.OK, response = TeamDTO.class)
+    })
+    @GetMapping
+    public ResponseEntity<List<TeamDTO>> findAll() {
+        return ResponseEntity.status(HttpStatus.OK).body(teamService.findAll().stream().map(teamMapper::toDto)
```

Comment:
Can you move logic of converting entity to dto in service ?

Reply:
You think it should be there? Our tech-expert didn`t told us anything when he had reviewed our code.